next.config.mjs

```
@@ -2,80 +2,45 @@
-    // See the following for more information:
-    // https://docs.sentry.io/product/crons/
-    // https://vercel.com/docs/cron-jobs
-    automaticVercelMonitors: true,
-  }
+export default bundleAnalyzer(
+  withNextIntlConfig({
+    transpilePackages: ["@heroui/react", "@heroui/theme"],
+    swcMinify: true,
+    eslint: {
+      ignoreDuringBuilds: true,
+    },
```

Biome (1.9.4)

[error] This property value named eslint is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named eslint


(lint/suspicious/noDuplicateObjectKeys)